configs/ote/custom-object-detection/gen3_mobilenetV2_SSD/model.py

```diff
@@ -18,6 +18,7 @@
         anchor_generator=dict(
             type='SSDAnchorGeneratorClustered',
             strides=(16, 32),
+            reclustering_anchors=True,
```

Comment:
It's a bit strange to have this parameter here, because it does not influence the bbox head itself. It's just a weird way to pass parameter to the training utility. Do we really need it here?